### PR TITLE
fixed duplicate container parameter

### PR DESCRIPTION
**Pull request by IvanKirpichnikov**

### CODE REVIEW

File: src/dishka/integrations/aiogram.py [hunk 1]
```diff
@@ -23,11 +23,14 @@
 
 
 def inject(func: Callable[P, T]) -> Callable[P, T]:
-    additional_params = [Parameter(
-        name=CONTAINER_NAME,
-        annotation=Container,
-        kind=Parameter.KEYWORD_ONLY,
-    )]
+    if CONTAINER_NAME in signature(func).parameters:
```
Comment (Tishka17) [inline src/dishka/integrations/aiogram.py:26]: Can we write a test?

Comment (IvanKirpichnikov) [reply src/dishka/integrations/aiogram.py:26]: Yes, I will do it

Comment (IvanKirpichnikov) [reply src/dishka/integrations/aiogram.py:26]: done

**Pull request merged**